Templates/AppSource App/.github/workflows/PullRequestHandler.yaml

```diff
@@ -16,74 +16,240 @@ permissions:
   pull-requests: read
 
 jobs:
-  PullRequestHandler:
+  PregateCheck:
```

Comment:
Shouldn't this job simply have `if: ${{ github.event.pull_request.base.repo.full_name != github.event.pull_request.head.repo.full_name}}` ?

Reply:
Good point :) Thank you

Reply:
Use 'If not failed'